src/components/Contributors.tsx

```diff
@@ -12,7 +12,12 @@ import { Dialog, DialogContent, DialogTrigger } from "./ui/dialog";
 import { CircleAlert } from "lucide-react";
 import { Button } from "./ui/button";
 import Link from "next/link";
+import dotenv from "dotenv";
```

Comment:
Remove this, you can access NEXT_EXAMPLE variable from env like process.env.NEXT_EXAMPLE. variable name can be anything this is just an example.